pubsub/azurepubsub/azurepubsub.go

```diff
@@ -435,6 +450,11 @@ func (s *subscription) SendAcks(ctx context.Context, ids []driver.AckID) error {
 		return nil
 	}
 
+	// Invoke AckFunc is provided (at-most-once mode).
```

Comment:
No, you don't have to do this, `pubsub.go` will. You can write 

```
if s.opts != nil && s.opts.AckFuncForReceiveAndDelete != nil {
  panic("unreachable")
}
```

if you want to.

Reply:
done